crates/polars-pipe/Cargo.toml

```diff
@@ -31,6 +31,9 @@ smartstring = { workspace = true }
 [build-dependencies]
 version_check = { workspace = true }
 
+[dev-dependencies]
```

Comment:
I checked and this is already a requirement for some of the other crates. My guess is the Python tests would be a lot slower? And this class isn't really exposed...

Reply:
No, the python test run all in a single compilation. Every unit test in rust is compiled in a seperate `main` and costs a lot of time. Most of the time is in compilation time for our CI.

Reply:
Oh, but back to my original issue (thinking out loud):

1. This class isn't exposed to Python. It could be purely for testing purposes, I suppose... but that's making the public Python API larger with a private API. I guess could be prefixed with `_`.
2. Some sort of end-to-end test. This is... possible, I guess. vstack the DataFrames, turn into a `LazyFrame`, do `collect(streaming=False)` and then check that the chunks are smaller.

Do you have a preference?

Reply:
Also, are you sure about the fact that it's binary per `#[test]`? Reading https://matklad.github.io/2021/02/27/delete-cargo-integration-tests.html suggests that _integration tests_ do indeed get a binary per test, and discusses how to fix that, but _unit tests_ are a single binary per crate so the linking cost is one-time cost per crate.

Reply:
Ok, but I am still not convinced we should do property tests on the rust side. Our CI isn't ready for this. We like to run fast unit-tests on our normal test runs and paramteric tests individually.

I can see use adding property tests on the rust side, but I want that set up then in CI properly first. Can you rewrite that test you wrote for only 3 inputs now and then in a seperate PR we can think of introducing property tests.

Reply:
Will do.